src/Call.tsx

```diff
@@ -896,21 +897,41 @@ export class Call extends EventEmitter {
    *    - Rejects when the message is unable to be sent.
    */
   async sendMessage(message: CallMessage): Promise<OutgoingCallMessage> {
+    if (!(message instanceof CallMessage)) {
+      throw new InvalidArgumentError(
+        'Argument "message" must be instanceof "CallMessage"'
+      );
+    }
+
+    if (
+      typeof message.getContent() === 'undefined' ||
+      message.getContent() === null
+    ) {
+      throw new InvalidArgumentError('"content" is empty');
+    }
+
+    //@ts-ignore
```

Comment:
Why do we need to `ts-ignore` here?

Reply:
oops left over from previous implementation, removed 👍

Reply:
It's because this case is type protected. `Property 'length' does not exist on type 'MessageType'.`. Since `MessageType` is an enum. Should I remove this error case?

Reply:
done

also 5.xx upgrade ticket: https://issues.corp.twilio.com/browse/VBLOCKS-2673